src/main/resources/userRecoveryEmailTemplateFragment-feedbackSessionAccessLink.html

```diff
@@ -0,0 +1,56 @@
+<br><table style="max-width:600px;border:1px solid black;">
```

Comment:
so basically the whole thing now looks like this:
![image](https://user-images.githubusercontent.com/28526038/53478178-32f43b80-3ab1-11e9-87a0-35e92fe77e8e.png)
![image](https://user-images.githubusercontent.com/28526038/53478338-85cdf300-3ab1-11e9-90e5-8b90ed468db0.png)

and one session has one table of 4 rows. Not sure if it is preferred to hv a single table containing all sessions and their details. The current design looks neat but for students with a lot of sessions might not be a good idea. Let's ask for some help here @damithc

Reply:
Yes, the current format seems to be too verbose. Why not one row per session, one table per course? e.g.,

Course: **CS2103 Software Engineering**

Session | links
--------|----
Peer evaluation 1 | [[submission link]()] [[results link]()]
Peer evaluation 2 | [[submission link]()]